taskcluster/ci/generate-screenshots/kind.yml

```diff
@@ -0,0 +1,50 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: taskgraph.loader.transform:loader
+
+transforms:
+    - ffios_taskgraph.transforms.screenshots:transforms
+    - ffios_taskgraph.transforms.secrets:transforms
+    - taskgraph.transforms.job:transforms
+    - taskgraph.transforms.task:transforms
+
+
+job-defaults:
+    description: Generate localized screenshots by delegating the work to bitrise.io
+    run:
+        using: run-commands
+        use-caches: false
+        secrets:
+            by-level:
+                '3':
+                    - name: project/mobile/firefox-ios/bitrise
+                      key: api_key
+                      path: .bitrise_token
+                default: []
+        dummy-secrets:
+            by-level:
+                '3': []
+                default:
+                      - content: "faketoken"
+                        path: .bitrise_token
+    run-on-tasks-for: [github-pull-request]
+    worker:
+        artifacts:
+            - type: file
+              name: public/logs/bitrise.log
+              path: /builds/worker/checkouts/src/bitrise.log
```

Comment:
This task downloads the actual bitrise logs and re-expose them in Taskcluster. Shell colors are even supported, check this out! https://firefox-ci-tc.services.mozilla.com/tasks/dFVHYucpRjOJEcdQp_OVNQ/runs/0/logs/https%3A%2F%2Ffirefox-ci-tc.services.mozilla.com%2Fapi%2Fqueue%2Fv1%2Ftask%2FdFVHYucpRjOJEcdQp_OVNQ%2Fruns%2F0%2Fartifacts%2Fpublic%2Flogs%2Fbitrise.log